### DISCUSSION TITLE
Cherry picking of characters in Sheet

### DISCUSSION THREAD

**Discussion by LeCuay** [EddieDover/fvtt-party-sheet#54] [Ideas]
# Summary

The party sheet is awesome for an overall view but for temporal allies, Animal Companion, Familiars and other NPCs you either need to change the sheet and manually discard **all** the NPCs except for the ones you need or you just leave them out.

If the configuration allowed some kind of cherry picking by character it'd be awesome. I know each sheet has its own data but many NPCs share the same base data as plain PCs -> AC, BAB, Saving throws...

**Comment by EddieDover** [comment]
I'm not against this, we just need to come up a way to specify the cherry picked entity.

**Comment by tjlanza** [comment]
Honestly, I was surprised to see the choices are "show only online" or "show-everybody-but". I would have expected to be able to pick which Actors are show.

**Reply by EddieDover** [reply]
> Honestly, I was surprised to see the choices are "show only online" or "show-everybody-but". I would have expected to be able to pick which Actors are show.

Because the original intent of this module was to show the actual player's party data, not any other NPCs, familiars, etc. The features that exist now are there because of user request. I try to make sure that if something is a big ask, I implement it if multiple people are interested in its use, or I just happen to be doing something adjacent code-wise.

Sometimes DMs want to see all of their players, even if only a few are online, hence “show only online” being a toggle. Toggling out specific actors stemmed from other game systems where there was no difference between “pc” and “npc” and DMs needed a way to blacklist a small amount of actors rather than whitelist a larger amount of actors.

Now that I'm migrating everything over to V13 and dropping support for V12, I may have time, and be able, to revisit that selection dialog and have a 'blacklist/whitelist' toggle, but would happen after the next release.
